Source/WebCore/svg/SVGTextPathElement.cpp

```diff
@@ -168,6 +168,11 @@ void SVGTextPathElement::buildPendingResource()
     if (!renderer)
         return;
 
+    if (document().settings().layerBasedSVGEngineEnabled()) {
+        renderer->repaint();
```

Comment:
Repainting here does not seem the right thing here. Does it work if this just early returns?

Reply:
@rwlbuis - Yes - it works without repaint as well. So just pushed with simple early return.